assets/dev/js/editor/components/browser-import/session.js

```diff
@@ -109,7 +109,7 @@ export default class Session {
 				case 'container':
 				case 'section':
 				case 'column':
-				case 'div-block':
+				case 'e-div-block':
```

Comment:
Please create a doc file with all the places we use `div-block` so we will later delete it